### ISSUE TITLE
Menu bar color 

### ISSUE THREAD

**Issue by RafalSlomka** [microsoft/vscode#193008]
Type: <b>Bug</b>

After update to v1.82.0 menu bar changed color to gray when the whole theme is dark. This contrast is a bit annoying :) How can it be changed back to dark?
In settings.json -> "workbench.colorCustomizations" I can see only "menubar.selection*" options.

VS Code version: Code 1.82.0 (8b617bd08fd9e3fc94d14adb8d358b56e3f72314, 2023-09-06T22:07:18.759Z)
OS version: Linux x64 5.4.0-150-generic
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz (4 x 2592)|
|GPU Status|2d_canvas: unavailable_software<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>opengl: disabled_off<br>rasterization: disabled_software<br>raw_draw: disabled_off_ok<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software<br>webgpu: disabled_off|
|Load (avg)|1, 1, 1|
|Memory (System)|10.71GB (2.27GB free)|
|Process Argv|--unity-launch --crash-reporter-id a915539e-8863-4943-a922-512efd3a793e|
|Screen Reader|no|
|VM|67%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (47)</summary>

Extension|Author (truncated)|Version
---|---|---
vagrant|bbe|0.5.0
systemd-unit-file|coo|1.0.6
doxdocgen|csc|1.4.0
yash|dao|0.2.9
vscode-eslint|dba|2.4.2
vscode-pull-request-github|Git|0.72.0
go|gol|0.39.1
gc-excelviewer|Gra|4.2.58
todo-tree|Gru|0.0.226
githd|hui|2.3.3
better-cpp-syntax|jef|1.17.2
cmake-language-support-vscode|jos|0.0.9
rpm-spec|Lau|0.3.2
jsonnet-language|lia|1.0.0
autoconf|mae|0.2.0
dotenv|mik|1.0.1
vscode-docker|ms-|1.26.0
vscode-dotnet-runtime|ms-|1.7.2
isort|ms-|2023.10.1
python|ms-|2023.16.0
vscode-pylance|ms-|2023.9.10
jupyter|ms-|2023.8.1002501831
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.17
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.309.0
cmake-tools|ms-|1.15.31
cpptools|ms-|1.17.5
cpptools-extension-pack|ms-|1.3.0
makefile-tools|ms-|0.7.0
go-lazy-mock|naw|1.0.3
vscode-protolint|ple|0.8.0
java|red|1.22.0
vscode-go-impl-methods|ric|1.1.1
scala|sca|0.5.7
cmake|twx|0.0.17
sort-lines|Tyr|1.10.2
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.2.30
vscode-java-debug|vsc|0.54.0
vscode-java-dependency|vsc|0.23.1
vscode-java-pack|vsc|0.25.14
vscode-java-test|vsc|0.39.1
vscode-maven|vsc|0.42.0
markdown-all-in-one|yzh|3.5.1
vscode-proto3|zxh|0.5.5

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
vscccc:30803845
282f8724:30602487
89544117:30613380
2i9eh265:30646982
showlangstatbar:30737416
0bi6i642:30835152
a2ce3375:30757347
pythonfmttext:30731395
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
pythonnosmt12:30797651
pythonidxptcf:30805731
pythonnoceb:30805159
dsvsc013:30795093
dsvsc014:30804076
diffeditorv1:30821571
dsvsc015cf:30829746

```

</details>

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.82.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by RafalSlomka** [comment]
Issue remains after update to 1.82.1.

**Comment by ArturoDent** [comment]
Are you referring to `titleBar.activeBackground` perhaps?

**Comment by RafalSlomka** [comment]
I'm referring to this gray bar. I'd call it menu bar, but maybe it's called title bar. Is it?
![image](https://github.com/microsoft/vscode/assets/31282482/045b8a93-507f-4ec4-b3ee-cdf72989b85c)

**Comment by aeschli** [comment]
This looks like the native window title bar and the color is configured in the OS. Depends on your OS. e.g.https://unix.stackexchange.com/questions/595957/how-to-change-the-color-of-an-active-window-title-bar-in-gnome

Why the color changed I can not explain.

VS Code uses the native window title bar in Linux by default, but you can also switch to the custom one: `window.titleBarStylewindow.titleBarStyle` is the setting. Then the VS Code color `titleBar.activeBackground` is used.

**Comment by MattRyder** [comment]
I encountered this recently. I had `window.titleBarStyle` set to `native`, and I saw the light-coloured title bar after an update. 
I fixed it by switching it to `custom`, which prompts a restart, then going back and returning the value back to `native`. Restarting again returns me back to the expected view.

Hope that helps anyone else that encounters this.

![image](https://github.com/microsoft/vscode/assets/840351/3ef31d20-f15f-4a78-a342-2d7d45e2d1f1)


**System Info**
> Version: 1.83.1
> Commit: f1b07bd25dfad64b0167beb15359ae573aecd2cc
> Date: 2023-10-12T12:50:57.295Z
> Electron: 25.9.1
> ElectronBuildId: undefined
> Chromium: 114.0.5735.289
> Node.js: 18.15.0
> V8: 11.4.183.29-electron.0
> OS: Linux x64 6.5.5-arch1-1
